app.feeddeck.feeddeck.yml

```diff
@@ -0,0 +1,46 @@
+app-id: app.feeddeck.feeddeck
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: feeddeck
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
```

Comment:
Why do you need home access?

Reply:
The app uses Supabase, which stores the users information in the `home` directory. If this permission is not set, a users session is not persisted and the user must login again as soon as the app is restarted.

Reply:
> in the home directory

Where?

Sounds like it should use the standard `XDG_` variables for storing.

Reply:
It's stored in `~/Documents/auth`.

Should this be changed to `- --filesystem=xdg-documents`?

Reply:
Yea probably. Unless you need it for anything else.

Reply:
I changed it. I think it's not needed for sth. else, at least it is also working when trying it locally.